DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -243,7 +243,10 @@
   <ioread sourceClass = "pat::Jet" version="[1-11]" targetClass="pat::Jet" source="int partonFlavour_" target="jetFlavourInfo_">
     <![CDATA[jetFlavourInfo_ = reco::JetFlavourInfo(0,onfile.partonFlavour_);]]>
   </ioread>
-  <class name="pat::MET"  ClassVersion="15">
+  <class name="pat::MET"  ClassVersion="18">
+   <version ClassVersion="18" checksum="428901429"/>
```

Comment:
What are ClassVersion 16 and 17 for?

Reply:
as we were changing the code, there were a couple of times we edited the pat MET class, and to get to compile we had to increment it. please let us know if need to fix it some other way

Reply:
Please cancel the intermediate ones and put classversion 16 in the most recent one